rdb/src/main/java/com/instana/dc/rdb/impl/informix/InformixUtil.java

```diff
@@ -12,38 +12,41 @@ private InformixUtil() {
+    public static final String INSTANCE_COUNT_SQL = "SELECT COUNT(DISTINCT DBSERVERNAME) AS SERVER FROM SYSTABLES;";
+    public static final String INSTANCE_ACTIVE_COUNT_SQL = "SELECT COUNT(DISTINCT name) AS ACTIVE_SERVER FROM SYSCLUSTER WHERE server_status = 'Active';";
 
     //Session & Active Session information (KPI)
-    public static final String ACTIVE_SESSION = "select count(1) from syssessions;";
-    public static final String SESSION_COUNT_SQL = "select count(1) from syssessions;";
+    public static final String ACTIVE_SESSION = "SELECT COUNT(1) FROM SYSSESSIONS;";
+    public static final String SESSION_COUNT_SQL = "SELECT COUNT(1) FROM SYSSESSIONS;";
 
 
-    //I/O Read & Write information (KPI) : As number of Reads/Writes PER CHUNK hence doing the sum
-    public static final String IO_READ_COUNT_SQL = "SELECT SUM(syschunks.pagesize * syschkio.pagesread) FROM SYSCHKIO INNER JOIN syschunks ON syschunks.chknum=syschkio.chunknum;";
-    public static final String IO_WRITE_COUNT_SQL = "SELECT SUM(syschunks.pagesize * syschkio.pageswritten) FROM SYSCHKIO INNER JOIN syschunks ON syschunks.chknum=syschkio.chunknum;";
+    //I/O Read & Write information (KPI) : As number of Reads/Writes PER CHUNK hence doing the SUM
+    public static final String IO_READ_COUNT_SQL = "SELECT SUM(SYSCHUNKS.pagesize * SYSCHKIO.pagesread) FROM SYSCHKIO INNER JOIN SYSCHUNKS ON SYSCHUNKS.chknum=SYSCHKIO.chunknum;";
+    public static final String IO_WRITE_COUNT_SQL = "SELECT SUM(SYSCHUNKS.pagesize * SYSCHKIO.pageswritten) FROM SYSCHKIO INNER JOIN SYSCHUNKS ON SYSCHUNKS.chknum=SYSCHKIO.chunknum;";
 
-    public static final String MEMORY_UTILIZATION_SQL = "SELECT (sum(seg_blkused) * 100) / (sum(seg_blkused) + sum(seg_blkfree)) FROM SYSSEGLST;";
+    public static final String MEMORY_UTILIZATION_SQL = "SELECT (SUM(seg_blkused) * 100) / (SUM(seg_blkused) + SUM(seg_blkfree)) FROM SYSSEGLST;";
 
-    public static final String SQL_COUNT_SQL = "SELECT count(1) from syssqltrace where (dbinfo('utc_current') - sql_finishtime)<24*60*60;";
-    public static final String TRANSACTION_COUNT_SQL = "select count(1) from systrans;";
-    public static final String SQL_ELAPSED_TIME_SQL = "SELECT sql_runtime*1000 as ELAPSED_TIME_MILLIS, sql_id as sql_id, sql_statement as sql_text FROM sysmaster:syssqltrace  where sql_statement not like '%syssqltrace%' ORDER BY sql_runtime desc limit 20;";
+    public static final String SQL_COUNT_SQL = "SELECT COUNT(1) FROM SYSSQLTRACE WHERE (dbinfo('utc_current') - sql_finishtime)<24*60*60;";
+    public static final String TRANSACTION_COUNT_SQL = "SELECT COUNT(1) FROM SYSTRANS;";
+    public static final String SQL_ELAPSED_TIME_SQL = "SELECT sql_runtime*1000 AS ELAPSED_TIME_MILLIS, sql_id AS sql_id, sql_statement AS sql_text FROM SYSMASTER:SYSSQLTRACE  WHERE sql_statement NOT LIKE '%syssqltrace%' ORDER BY sql_runtime DESC LIMIT 20;";
```

Comment:
I believe we can have more filter for this query and please validate the usage of `sql_statement NOT LIKE '%syssqltrace%'` also.

Reply:
Updated.